package.json

```diff
@@ -18,15 +22,16 @@
     "type": "git",
     "url": "git+https://github.com/gabrielcsapo/node-git-server.git"
   },
-  "main": "index.js",
+  "main": "dist/index.js",
   "engine": {
     "node": ">= 14"
   },
   "scripts": {
     "lint": "eslint src/*",
     "test": "jest",
-    "build": "tsc",
-    "coverage": "jest --coverage"
+    "build": "echo \"\" > ./build-complete.meta && tsc",
+    "coverage": "jest --coverage",
+    "install": "( [ -f ./git-package.meta ] && [ ! -f ./build-complete.meta ] && echo \"\" > ./build-complete.meta && npm install . && tsc ) || echo \"\" > ./build-complete.meta"
```

Comment:
what does this do?

Reply:
For some reason npm does not build any package installed from git, so I added this cos it was too annoying (however I forgot to delete `dist` while testing, so I noticed that it doesn't work only when I committed this)